addons/sale_product_matrix/migrations/13.0.1.0/post-migration.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 Tecnativa - Carlos Roca
```

Comment:
In general terms I think we use the full version as folder name. This file should be in the `13.0.1.0.0` folder.

Reply:
That doesn't work. It should follow the exact manifest version.